<launch>
  <arg name="target_distance" default="1.0" />
  <param name="/follower_node/target_distance" value="$(arg target_distance)" type="double"/>
  
  <!--需要启动两个机器人相对于世界的坐标关系的发布-->
  <!--
      基本实现思路：
          1.节点只编写一次
          2.这个节点需要启动两次
          3.节点启动时动态传参：第一次传递tb3_0，第二次传递tb3_1
  -->
  <node  pkg="ros_package2" type="tb3_pub" name="pub0" args="tb3_0" output="screen" />
  <node  pkg="ros_package2" type="tb3_pub" name="pub1" args="tb3_1" output="screen" />
  
  <!--需要订阅tb3_0与tb3_1相对于世界坐标系的坐标消息，
      并转换成tb3_0相对于tb3_1的坐标关系
      再生成速度消息
  -->
  <node  pkg="ros_package2" type="follower_node" name="control0" output="screen" />
  <node  pkg="ros_package2" type="follower_node" name="control1" output="screen" />

</launch>
